src/diffusers/models/vq_model.py

```diff
@@ -142,21 +142,26 @@ def decode(
     ) -> Union[DecoderOutput, torch.FloatTensor]:
         # also go through quantization layer
         if not force_not_quantize:
-            quant, _, _ = self.quantize(h)
+            quant, commit_loss, _ = self.quantize(h)
         elif self.config.lookup_from_codebook:
             quant = self.quantize.get_codebook_entry(h, shape)
+            commit_loss = torch.zeros((h.shape[0])).to(h.device, dtype=h.dtype)
         else:
             quant = h
+            commit_loss = torch.zeros((h.shape[0])).to(h.device, dtype=h.dtype)
         quant2 = self.post_quant_conv(quant)
         dec = self.decoder(quant2, quant if self.config.norm_type == "spatial" else None)
 
         if not return_dict:
-            return (dec,)
+            return (
+                dec,
+                commit_loss,
+            )
```

Comment:
```suggestion
            return dec, commit_loss
```

Does this not break the existing workflows because we're changing the output signature.

Reply:
@sayakpaul I don't think so since currently, when sampling the decoded result, all the codes in diffusers seem to do
```
self.vae.decode(latents / self.vae.config.scaling_factor, return_dict=False)[0]
```
if return dict is not False. Since technically, the output type is 
```
Union[DecoderOutput, tuple[torch.FloatTensor]]
```
I think from the start